Capfile

```diff
@@ -7,9 +7,8 @@ require 'capistrano/deploy'
 # Include tasks from other gems included in your Gemfile
 require 'capistrano/bundler'
 require 'capistrano/rails/assets'
-require 'capistrano/rails/migrations'
+# require 'capistrano/rails/migrations'
```

Comment:
It was causing issues when I was deploying, although I'm not sure if I fixed them or not.

I'll give it another run with it back in there.